products/compute/terraform.yaml

```diff
@@ -302,6 +302,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         state as plain-text. [Read more about sensitive data in state](/docs/state/sensitive-data.html).
         Because the API does not return the sensitive key value,
         we cannot confirm or reverse changes to a key outside of Terraform.
+  DiskResourcePoliciesAttachment: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
You'll also likely need to add an `exclude: true` in `inspec.yaml` and `ansible.yaml` similar to https://github.com/GoogleCloudPlatform/magic-modules/pull/1504/files#diff-63dae33e5de228cd42cb0e6a772a7dbdR22-R23